articles/truebit_incentive/truebit_light_incentives.tex

```diff
@@ -120,35 +120,35 @@
 correct computations/transactions to be included.
 
 This article wants to specifically address the Dogecoin-Ethereum bridge, which requires blocks from the Dogecoin
-blockchain to be verified inside an Ethereum smart contract. This verification is too expensive to be done directly,
-and because of that, we utilize TrueBit to take the computation off-chain.
+blockchain to be verified inside an Ethereum smart contract. This verification is expensive when done directly,
+and because of that, we are incentivized to utilize TrueBit to take the computation off-chain.
```

Comment:
Hm, perhaps it is better to remove the "we" from the second part. Something like "it is beneficial to utilize...". What do you think?